react/features/base/domains/functions.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import { toState } from '../redux';
+
+/**
+ * A list of domains we consider jitsi-enabled domains by default. This is in
+ * line with the app links on iOS and Android, but stands here as retreiving
+ * those programatically is not straightforward.
+ */
+const JITSI_KNOWN_DOMAINS = [
+    'beta.meet.jit.si',
+    'beta.hipchat.me',
+    'chaos.hipchat.me',
+    'enso.me',
+    'hipchat.me',
+    'meet.jit.si'
+];
+
+/**
+ * Returns a list of default domains defined by the app. If the app received a
+ * knownDomains prop on instantiation it returns that, otherwise we use the
+ * jitsi defaults.
+ *
+ * @param {Object | Function} stateful - The redux store, state, or
+ * {@code getState} function.
+ * @returns {Array<string>}
+ */
+export function getDefaultDomains(stateful: Object | Function) {
+    const { knownDomains } = toState(stateful)['features/app'].app.props;
+
+    if (Array.isArray(knownDomains)) {
+        return knownDomains;
+    }
+
+    return JITSI_KNOWN_DOMAINS;
+}
+
+/**
+ * Returns the list of known domains that are managed by the app.
+ *
+ * NOTE: This function is a workaround to have the feature reducer and
+ * middleware registered as this is a passive feature, other features normally
+ * don't import anything from here, other than the content of its store itself.
+ *
+ * @param {Object | Function} stateful - The redux store, state, or
+ * {@code getState} function.
+ * @returns {Array<string>}
+ */
+export function getKnownDomains(stateful: Object | Function) {
```

Comment:
please don't.

Reply:
Can you please explain?

Reply:
Same as on the settings PR, let's not create these one-liner functions for accessing the state of a given feature.

Reply:
Oh, this one is with a different purpose. Please see the docs above

Reply:
I see. Instead of doing that, import the feature here: https://github.com/jitsi/jitsi-meet/blob/master/react/features/app/components/App.native.js#L22 That's how we handle this for all those other features.